test.js

```diff
@@ -23,6 +23,12 @@ const nesting = [
   '\'One "sentence". Two sentences.\''
 ].join('\n\n')
 
+const multiNesting = [
+  'A sentence “with ‘multi «mutli “nested ‘quotes’”»’”.', // correct case when {smart: ['“”', '‘’', '«»']} (TODO: REMOVE THESE COMMENTS!!!)
```

Comment:
```suggestion
  'A sentence “with ‘multi «mutli “nested ‘quotes’”»’”.',
```

Some finished suggestions, removing the comments